docs/docsite/rst/vmware/scenario_rename_vm.rst

```diff
@@ -0,0 +1,171 @@
+
+Now you need to supply the information about the existing virtual machine which will be renamed. For renaming virtual machine, ``vmware_guest`` module uses VMware UUID, which is unique across vCenter environment. This value is autogenerated and can not be changed. You will use ``vmware_guest_facts`` module to find virtual machine and get information about VMware UUID of the virtual machine.
+
+This value will be used input for ``vmware_guest`` module. Specify new name to virtual machine which conforms to all VMware requirements for naming conventions as ``name`` parameter. Also, provide ``uuid`` as the value of VMware UUID.
+
+What to expect
+--------------
+
+.. code-block:: yaml
+
+    {
+        "changed": true,
+        "instance": {
+            "annotation": "",
+            "current_snapshot": null,
+            "customvalues": {},
+            "guest_consolidation_needed": false,
+            "guest_question": null,
+            "guest_tools_status": "guestToolsNotRunning",
+            "guest_tools_version": "10247",
+            "hw_cores_per_socket": 1,
+            "hw_datastores": ["ds_204_2"],
+            "hw_esxi_host": "10.x.x.x",
+            "hw_eth0": {
+                "addresstype": "assigned",
+                "ipaddresses": [],
+                "label": "Network adapter 1",
+                "macaddress": "00:50:56:8c:b8:42",
+                "macaddress_dash": "00-50-56-8c-b8-42",
+                "portgroup_key": "dvportgroup-31",
+                "portgroup_portkey": "15",
+                "summary": "DVSwitch: 50 0c 3a 69 df 78 2c 7b-6e 08 0a 89 e3 a6 31 17"
+            },
+            "hw_files": ["[ds_204_2] old_vm_name/old_vm_name.vmx", "[ds_204_2] old_vm_name/old_vm_name.nvram", "[ds_204_2] old_vm_name/old_vm_name.vmsd", "[ds_204_2] old_vm_name/vmware.log", "[ds_204_2] old_vm_name/old_vm_name.vmdk"],
+            "hw_folder": "/DC1/vm",
+            "hw_guest_full_name": null,
+            "hw_guest_ha_state": null,
+            "hw_guest_id": null,
+            "hw_interfaces": ["eth0"],
+            "hw_is_template": false,
+            "hw_memtotal_mb": 1024,
+            "hw_name": "new_vm_name",
+            "hw_power_status": "poweredOff",
+            "hw_processor_count": 1,
+            "hw_product_uuid": "420cbebb-835b-980b-7050-8aea9b7b0a6d",
+            "hw_version": "vmx-13",
+            "instance_uuid": "500c60a6-b7b4-8ae5-970f-054905246a6f",
+            "ipv4": null,
+            "ipv6": null,
+            "module_hw": true,
+            "snapshots": []
+        }
+    }
+
+- State is changed to ``True`` which notifies that the virtual machine is renamed to the given new name. This operation can take some time depending on your environment and network connectivity.
```

Comment:
Adding the playbook output would be useful here. Suggested re-wording:
I'd change this paragraph to:
"Running this playbook can take some time, depending on your environment and network connectivity. When the run is complete you will see 
[insert playbook output here]
confirming that you've renamed the virtual machine."